packages/text-bitmap/src/BitmapFont.ts

```diff
@@ -237,6 +237,41 @@ export class BitmapFont
         return available;
     }
 
+    /**
+     * This character set includes all the letters in the alphabet (both lower- and upper- case).
+     * @readonly
+     * @static
+     * @member {string[][]}
+     * @example BitmapFont.from({ chars: [...BitmapFont.ALPHA], name: "ExampleFont" })
+     */
+    public static ALPHA = [['a', 'z'], ['A', 'Z']];
+
+    /**
+     * This character set includes all decimal digits (from 0 to 9).
+     * @readonly
+     * @static
+     * @member {string[][]}
+     * @example BitmapFont.from({ chars: [...BitmapFont.NUMERIC], name: "ExampleFont" })
+     */
+    public static NUMERIC = [['0', '9']];
+
+    /**
+     * This character set is the union of `BitmapFont.ALPHA` and `BitmapFont.NUMERIC`.
+     * @readonly
+     * @static
+     * @member {string[][]}
+     */
+    public static ALPHANUMERIC = [['a', 'z'], ['A', 'Z'], ['0', '9']];
+
+    /**
+     * This character set consists of all the ASCII table.
+     * @readonly
+     * @static
+     * @member {string[][]}
+     * @see http://www.asciitable.com/
+     */
+    public static ASCII = [['\0', '~']];
```

Comment:
These are marked as readonly in the docs, but not in the Typescript, is that intentional?